docs/examples.md

```diff
@@ -1215,3 +1215,9 @@ automations:
 ```
 
 ![Zoom automation example](images/zoom-automation.gif 'Zoom automation example :size=400')
+
+## Doorbell
```

Comment:
The examples are intended to be sorted alphabetically. Mind moving it up? Thank you!

Reply:
Ops, got it. Will move.